qbft/commit.go

```diff
@@ -94,7 +95,7 @@ func CreateCommit(state *State, config IConfig, root [32]byte) (*SignedMessage,
 	return signedMsg, nil
 }
 
-func baseCommitValidation(
+func baseCommitValidationNoVerification(
```

Comment:
the config is not used here anymore, should we delete it?

Reply:
Indeed. I'll remove it. Thanks

Reply:
Yeah. "IgnoreSignature" is more clear. I'll update it. Thanks!